src/LocalCoords.h

```diff
@@ -73,6 +73,12 @@ class LocalCoords {
   /** A Point representing the 3D coordinates of this LocalCoords */
   Point _coords;
 
+  /** The direction angle in radians with respect to the x-axis */
+  double _phi;
```

Comment:
Need to define functions to set the direction values

Reply:
added